docs/03-dev-howtos/16-working-with-amp.md

```diff
@@ -64,5 +64,5 @@ This change was made to improve the experience of developing and styling amp pag
 
 ## Ads in Amp
 
-* [Prebid](https://prebid.org/) in Amp uses a Prebid server and is documented in a separate repo [here](https://github.com/guardian/prebid-server/tree/master/amp).
+* [Prebid](https://prebid.org/) in Amp uses a Prebid server and is documented in a separate repo [here](https://github.com/guardian/prebid-server/tree/main/amp).
```

Comment:
Perhaps this should be left out until the prebid-server repo is switched to main?